guide/payments/coin-selection.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: guide
+title: Coin Selection
+nav_order: 1
+parent: Payments
+permalink: /guide/payments/coin-selection/
+---
+
+{% include picture.html
+
+  image = "/assets/images/payments/coin-selection-header.png"
+
+  retina = "/assets/images/payments/coin-selection-header@2x.png"
+
+  mobile = "/assets/images/payments/coin-selection-header.png"
+
+  mobileRetina = "/assets/images/payments/coin-selection-header@2x.png"
+
+  alt-text = ""
+
+  width = 1600
+
+  height = 900
+
+  layout = "full-width"
+
+%}
+
+# Coin Selection
+
+## Intro
+
+Coin selection is the process of choosing which of your Bitcoins to spend when creating and approving a spending transaction. To understand coin selection a little better, we must first grasp the concept of Unspent Transaction Outputs (UTXOs).
+
+A UTXO is the amount of digital currency remaining after a bitcoin transaction has been executed - essentially, the unspent change left after someone completes a transaction, similar to the change someone receives after conducting a cash transaction at the store. They are processed continuously and are responsible for beginning and ending each bitcoin transaction. They might sound complicated, but they really are quite simple. 
```

Comment:
@johnsBeharry is there any other part of the payment guide that explains UTXOs? I feel like this can be referenced to somewhere instead of repeated?